libs/sql-schema-describer/tests/postgres_describer_tests.rs

```diff
@@ -11,6 +11,29 @@ use sql_schema_describer::*;
 use test_api::*;
 use test_macros::test_each_connector;
 
+/*
+This doesn't seem to be possible...
+'executing migration: Error { kind: QueryError(Error { kind: Db, cause: Some(DbError { severity: "ERROR", parsed_severity: Some(Error), code: SqlState("42601"), message: "syntax error at or near \"PROCEDURE\"", detail: None, hint: None, position: Some(Original(17)), where_: None, schema: None, table: None, column: None, datatype: None, constraint: None, file: Some("scan.l"), line: Some(1127), routine: Some("scanner_yyerror") }) }), original_code: Some("42601"), original_message: Some("db error: ERROR: syntax error at or near \"PROCEDURE\"") }', libs/sql-schema-describer/tests/postgres/mod.rs:30:31
+#[tokio::test]
+async fn procedures_can_be_described() {
+    let db_name = "procedures_can_be_described";
+
+    let sql = format!(
+        r#"
+        CREATE PROCEDURE "{}".foo(a integer, b integer) AS SELECT a, b;
+        "#,
+        SCHEMA
+    );
+
+    let inspector = get_postgres_describer(dbg!(&sql), db_name).await;
+    let result = inspector.describe(SCHEMA).await.expect("describing");
+    let procedure = result.get_procedure("foo").unwrap();
+
+    assert_eq!("foo", &procedure.name);
+    assert_eq!("SELECT 1 INTO res", &procedure.definition);
+}
+*/
```

Comment:
We're fine not being able to define them for now, imo

Reply:
Although it would be nice to figure out

Reply:
Works well from datagrip...